sunpy/io/tests/test_fits.py

```diff
@@ -138,7 +138,8 @@ def test_fitsheader():
     """Test that all test data can be converted back to a FITS header."""
     extensions = ('.fts', '.fits')
     for ext in extensions:
-        test_files = [f for f in get_test_data_filenames() if f.suffix == ext]
+        bad_file = "aia_lev1_211a_2022_03_30t17_21_21_63z_image_lev1.fits"
```

Comment:
```suggestion
        bad_file = "not_actually_fits.fits"
```
When the file has been renamed.